lib/notifications/notification_server.ex

```diff
@@ -36,6 +36,29 @@ defmodule Notifications.NotificationServer do
     GenServer.cast(server, {:bridge_movement, bridge_movement})
   end
 
+  @doc """
+  Creates a new Detour Activated Notification for a `Skate.Detours.Db.Detour`.
+
+  ## Options
+
+  If the `:server` option is present, the notification is sent to the process
+  referred to by the `:server` value.
+
+  If the `:notify_finished` option is present, a `{:new_notification, detour: detour.id}` message
+  is sent to the process referred to by the `:notify_finished` value.
+  """
+  @spec detour_activated(detour :: Skate.Detours.Db.Detour.t()) :: :ok
+  @spec detour_activated(detour :: Skate.Detours.Db.Detour.t(), keyword()) :: :ok
```

Comment:
This is so silly, but I didn't not realize until now that you can stack multiple specs for conditional parameters 😅 I've needed that before! That said, since you provide a default value for options, I don't think it's conditional anymore, so can the top spec be eliminated?

Reply:
Removed!